Modules/Core/Mesh/test/itkMeshTest.cxx

```diff
@@ -231,11 +229,9 @@ itkMeshTest(int, char *[])
 
     // Check if right cells are recovered
     index = 0;
-    for (std::map<itk::CellGeometryEnum, unsigned int>::iterator iter = cellPointMap.begin();
-         iter != cellPointMap.end();
-         ++iter)
+    for (auto & iter : cellPointMap)
     {
-      const unsigned int numOfPoints = iter->second;
+      const unsigned int numOfPoints = iter.second;
```

Comment:
Here, again, `iter` is not an iterator!!! Maybe just rename it to `pair`, because that's what it certainly is:

    for (auto & pair : cellPointMap)...

Reply:
Done in 2 places.